dev-support/docker/pkglist

```diff
@@ -33,7 +33,22 @@ openjdk-8-jdk
 python3-setuptools
 python3-pip
 python3.6
+python3.6-dev
+python3.6-venv
```

Comment:
Should we drop 3.6? (also note python is aliased to python3.6 in dev-support/docker/Dockerfile)

Reply:
it's to be dropped in Beam 2.36.0.

Reply:
I see, that makes sense. I thought maybe we were already dropping it because of the `for ver in 3.7 3.8 3.9; do` in `local-env-setup.sh`